src/identity/Identity.ts

```diff
@@ -107,7 +107,7 @@ export default class Identity {
       throw SDKErrors.ERROR_MNEMONIC_PHRASE_INVALID()
     }
 
-    const seed = toSeed(phrase)
+    const seed = mnemonicToMiniSecret(phrase)
```

Comment:
`toSeed` was removed in [v2.12.1](https://github.com/polkadot-js/common/releases/tag/v2.12.1).